extract_section_boundaries.py

```
@@ -0,0 +1,125 @@
+    # 2. セクション内容を分析
+    section_part = body[outside_start : outside_start + 3000]  # 3000文字程度を確認
+
+    # 3. 各種セクションマーカーを探す
+    markers_to_find = [
+        "🧹 Nitpick comments",
+        "🔇 Additional comments",
+        "</blockquote></details>",
+        "> </blockquote></details>",
+    ]
+
+    print(f"\n📊 セクション内のマーカー位置:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)